packages/react/src/components/StructuredList/StructuredList.stories.js

```diff
@@ -105,7 +105,7 @@ Default.argTypes = {
 export const Selection = (args) => {
   const structuredListBodyRowGenerator = (numRows) => {
     return Array.apply(null, Array(numRows)).map((n, i) => (
-      <StructuredListRow key={`row-${i}`}>
+      <StructuredListRow key={`row-${i}`} aria-busy="true">
```

Comment:
Do you think this should this be included in the component source instead of just the stories, or no?

Reply:
**100%** I don't why I didn't put it there 😅